prep-html-syntax/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <head>
```

Comment:
Be sure to structure your skeleton with the `<head> </head>` open and closed before the `<body>` opens.

Reply:
![R2 screenshot](https://user-images.githubusercontent.com/96153171/148866384-f7ff12ed-093f-45bd-8a44-ac4f553803dd.png)

added head tags before opening the body